src/chocolatey/infrastructure.app/services/ChocolateyPackageService.cs

```diff
@@ -722,6 +722,8 @@ private IEnumerable<ChocolateyConfiguration> get_packages_from_config(string pac
                     if (pkgSettings.IgnoreDependencies) packageConfig.IgnoreDependencies = true;
                     if (pkgSettings.ApplyInstallArgumentsToDependencies) packageConfig.ApplyInstallArgumentsToDependencies = true;
                     if (pkgSettings.ApplyPackageParametersToDependencies) packageConfig.ApplyPackageParametersToDependencies = true;
+                    if (pkgSettings.Force) packageConfig.Force = true;
+                    packageConfig.CommandExecutionTimeoutSeconds = string.IsNullOrWhiteSpace(pkgSettings.Timeout.ToString()) ? packageConfig.CommandExecutionTimeoutSeconds : pkgSettings.Timeout;

```

Comment:
Just as a note, this won't work as you expect it though.
The `Timeout` even when converted to a string will never be null or empty.
The default of an integer in `0` and not empty (even when converted to a string).

Reply:
fixed, I set a default value for pkgSettings.Timeout and used it for comparison. I used -1 as default value is 0 is a valid timeout.

Reply:
It looks like you made a small mistake, unfortunately.
You set the default on the `Force` Parameter instead of the `Timeout` parameter.

Reply:
@AdmiringWorm  oops that was a mistake, it should be fixed now.